diff -up dejagnu-1.6.3/runtest.exp.1~ dejagnu-1.6.3/runtest.exp
--- dejagnu-1.6.3/runtest.exp.1~	2022-03-17 21:21:59.168758987 +0100
+++ dejagnu-1.6.3/runtest.exp	2022-03-17 21:22:43.248613620 +0100
@@ -1601,8 +1601,7 @@ proc runtest { test_file_name } {
     global testdir
 
     clone_output "Running $test_file_name ..."
-    set prms_id	0
-    set bug_id	0
+    reset_vars
     set test_result ""
 
     # set testdir so testsuite file -test has a starting point
